allocations/tasks.py

```diff
@@ -247,12 +256,20 @@ def active_approved_allocations():
                     for c in allocation_charges:
                         if c.end_time > now:
                             _fork_charge(c, now, alloc)
+                LOG.info(f"Activating allocation {alloc.id} for project {charge_code}")
                 alloc.status = "active"
                 alloc.save()
-                KeycloakClient().update_project(
+                keycloak_client = KeycloakClient()
+                keycloak_client.update_project(
                     charge_code, has_active_allocation="true"
                 )
 
+                updated_project = keycloak_client._lookup_group(charge_code)
```

Comment:
Thoughts on just moving the alloc.save() line into an an `if` using this data?

Reply:
I'm nervous about messing with behavior until we determine exactly what's going on. This whole thing is so fickle I don't want to potentially make things worse.